lib/src/main/java/scal/io/liger/Constants.java

```diff
@@ -35,6 +35,7 @@
     public static final String AUDIO = "audio";
     public static final String PHOTO = "photo";
     public static final String VIDEO = "video";
+    public static final String IMAGE = "image";
```

Comment:
Can you rename this and/or move it so it doesn't look like its a key like the other three?  really AUDIO/VIDEO/VIDEO should be renamed something like

MEDIUM_AUDIO
MEDIUM_PHOTO
MEDIUM_VIDEO

maybe fixing that would help keep it clear too